src/solver/hydro/management/daily.cpp

```diff
@@ -237,7 +237,7 @@ inline void HydroManagement::prepareDailyOptimalGenerations(Solver::Variable::St
     auto const& maxP = maxPower[Data::PartHydro::genMaxP];
     auto const& maxE = maxPower[Data::PartHydro::genMaxE];
 
-    auto const& valgen = *ValeursGenereesParPays[numSpace][z];
+    auto& valgen = ValeursGenereesParPays[numSpace][z];
```

Comment:
_Const_ vanished

Reply:
`valgen.HydrauliqueModulableQuotidien[day] = problem.Turbine[dayMonth];`
It's modified later in the same file, I don't know why it was declared const in the first place

Reply:
Maybe the pointer was const, not the value